lavalink/websocket.py

```diff
@@ -60,6 +67,17 @@ def connected(self):
                 await self._node._manager._node_connect(self._node)
                 asyncio.ensure_future(self._listen())
 
+                if not self._resuming_configured and self._resume_key is not None \
+                        and self._resume_timeout is not None:
+                    await self._send(op='configureResuming', key=self._resume_key, timeout=self._resume_timeout)
+                    self._resuming_configured = True
+
+                if self._message_queue:
+                    for message in self._message_queue:
```

Comment:
Is here worth implementing a create_task or try, except? So one error doesn't impact other messages?

Reply:
No reason for messages to not send unless websocket is closed, or bad internet, in either case the other messages probably wouldn't send too

Reply:
true